scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml

```diff
@@ -0,0 +1,212 @@
+          export want_node_roles=controller=2,compute=2,storage=3 ;
+          export want_ceph=1 ;
+          export want_dvr=1 ;
+          export networkingplugin=openvswitch ;
+          export networkingmode=vlan ;
+          export cephvolumenumber=1 ;
+          export scenario=\"scenario.yml\" ;
+          export commands=\"$commands\" "'
+          wget --no-check-certificate -Oqa_crowbarsetup.sh "$runner_url"
+          wget --no-check-certificate -Oscenario.yml "$scenario_url"
+          [ $UPDATEBEFOREINSTALL == "true" ] && export updatesteps="addupdaterepo runupdate"
+          timeout --signal=ALRM 240m bash -x -c ". qa_crowbarsetup.sh ; onadmin_runlist $commands"
+          ' || ret=$?
+          echo "mkphyscloud ret=$ret (before scenario)"
+          if [ "$ret" = "0" ]; then
+            # ----- Prepare the SBD setup:
+            cat > /tmp/sbd_prepare_$admin <<EOSCRIPT
+              # preparation of iSCSI
+              export ZYPP_LOCK_TIMEOUT=120
+              zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP1/ --non-interactive install python-sh
+              wget --no-check-certificate https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/iscsictl.py
+              chmod +x iscsictl.py
+              ./iscsictl.py --service target --host \$(hostname) --no-key
+              ./iscsictl.py --service initiator --target_host \$(hostname) --host controller1 --no-key
+              ./iscsictl.py --service initiator --target_host \$(hostname) --host controller2 --no-key
+              # preparation of SBD
+              SBD_DEV=\$(ssh controller1 echo '/dev/disk/by-id/scsi-\$(lsscsi -i | grep LIO | tr -s " " |cut -d " " -f7)')
+              ssh controller1 "zypper --non-interactive install sbd; sbd -d \$SBD_DEV create"
+              ssh controller2 "zypper --non-interactive install sbd"
+              # watchdog configuration
+              ssh controller1 modprobe softdog; echo softdog > /etc/modules-load.d/watchdog.conf
+              ssh controller2 modprobe softdog; echo softdog > /etc/modules-load.d/watchdog.conf
+              # take scenario yaml file and replace /dev/sda with the right thing:
+              sed -i "s|/dev/sda|\${SBD_DEV}|g" scenario.yml
+              # ----- End of SBD
+          EOSCRIPT
+            chmod +x /tmp/sbd_prepare_$admin
+            scp /tmp/sbd_prepare_$admin root@$admin:sbd_prepare
+            ssh root@$admin './sbd_prepare
+            ' || ret=$?
+          fi
+          if [ "$ret" = "0" ]; then
+            ssh root@$admin "
+            export cloud=$cloud ;
+            export TESTHEAD=$TESTHEAD ;
+            export cloudsource=$cloudsource ;
+            export nodenumber=$nodenumber ;
+            export hacloud=$hacloud ;
+            export clusterconfig=$clusterconfig ;
+            export want_ceph=1 ;
+            export want_dvr =1;
```

Comment:
nitpick there shouldn't be a space before the ```=```. For consistency reasons you might want to add one before the ```;```